apps/files/models.py

```diff
@@ -131,10 +131,15 @@ def get_mirror(self, addon, attachment=False):
         return posixpath.join(*map(smart_str, [host, addon.id, self.filename]))
 
     def get_url_path(self, src):
+        return self._make_download_url('downloads.file', src)
```

Comment:
it's just [some weird legacy convention](https://github.com/mozilla/olympia/blob/master/apps/amo/models.py#L350-L351)